.travis.yml

```diff
@@ -31,7 +31,6 @@ matrix:
   allow_failures:
     # temporary until we find out how to prevent (spurious) build failures
     - env: TARGET="docs"
-    - env: TARGET="linux"
```

Comment:
@arturoc I very much would prefer if we left the linux build in allow_failures, so that we can still get a feeling how often the build fails due to memory concerns, and if whatever measures we implement have a meaningful impact.